modules/ROOT/pages/runtime-configure-gov.adoc

```diff
@@ -0,0 +1,87 @@
+= To Configure the Mule Runtime to Connect to the Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To use Mule Runtime within the Government Cloud, you must redefine the configuration properties that enable Mule Runtime to point to the EU control plane. You must also provide the client ID and secret of your environment.
+
+By default, the Mule Runtime is configured to point to the US control plane. If you do not override these properties, the Mule Runtime modules are disabled in the EU control plane.
+
+The properties you must override depend on which version of the Mule runtime you are using, as described in the following sections.
```

Comment:
The properties you must override depend on the version of the Mule Runtime you are using....

Reply:
Good catch - thank you!